foxpuppet/windows/browser/panel_ui/panel_ui.py

```diff
@@ -104,6 +108,17 @@ def open_private_window(self) -> None:
                 lambda _: set(self.selenium.window_handles) - initial_handles,
                 message="Private window did not open",
             )
+            new_private_window = (
+                set(self.selenium.window_handles) - initial_handles
+            ).pop()
+            self.selenium.switch_to.window(new_private_window)
+
+            from foxpuppet.windows.browser.window import BrowserWindow
```

Comment:
ehh, I don't quite like this even though I know why you're doing it. Let's just keep the imports at the top if we can.

Reply:
This will cause a circular import error.

Reply:
Ignore me then